src/clj_kondo/impl/config.clj

```diff
@@ -13,6 +13,7 @@
     ;; :config-in-comment {} config override for comment blocks
     :linters {:invalid-arity {:level :error
                               :skip-args [#_riemann.test/test-stream]}
+              :repeated-arity {:level :error}
```

Comment:
naming suggestion: conflicting-fn-arity or conflicting-fn-overload
in my opinion it communicates the problem better: you can't have more than one, whereas repeated is more ambigious